packages/balancer/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { Chains } from './utils'
```

Comment:
This is going to make it super easy to upgrade to the internal utils package - nice